journals/settings/base.py

```diff
@@ -324,7 +324,7 @@ def root(*x): return join(abspath(PROJECT_ROOT), *x)
         'BACKEND': 'journals.apps.search.backend',
         'URLS': ['http://journals.elasticsearch:9200'],
         'INDEX': 'journals',
-        'TIMEOUT': 5,
+        'TIMEOUT': 60,
```

Comment:
A minute seems like a long timeout, but there doesn't seem to be a "best" number. 10 if the default if you don't specify. I would probably go somewhere between 10-30 to start and we can bump if needed. 

Other ways to minimize errors: change the 'OPTIONS' value to `{'max_retries': <val> ,  'retry_on_timeout': True}`

Reply:
good call, made that fix.
Also discovered another error with reading the document that I fixed as well.